docs/using-airbyte/getting-started/oss-quickstart.md

```diff
@@ -294,24 +294,25 @@ Ensure the security group configured for the EC2 Instance allows traffic in on t
 abctl local install --host [HOSTNAME]
 ```
 
-### Editing the Ingress
+### Suggested Resources
```

Comment:
Why this is under EC2 block? Can we move upper?

Reply:
Sure we can move it into a section of its own. I put it here because the majority of users who have had issues are running on a VM.